launch/global_mapper_nodelet.launch

```diff
@@ -15,11 +15,12 @@
 
       <!-- remaps -->
       <remap from="cloud_topic" to="/flight/xtion_depth/points" />
-      <remap from="~map_topic" to="~global_map" />
+      <remap from="~pixel_map_topic" to="~pixel_map" />
+      <remap from="~voxel_map_topic" to="~voxel_map" />
```

Comment:
Hadn't taken the time to do that yet, but done now.  7827984ee252af78387bc3e86fe347d385e31368.